js-packages/@fortawesome/fontawesome-free/scss/_list.scss

```diff
@@ -3,7 +3,7 @@
 
 .#{$fa-css-prefix}-ul {
   list-style-type: none;
-  margin-left: $fa-li-width * 5/4;
+  margin-left: $fa-li-width * 5*0.25;
```

Comment:
Can be simplyfied to `margin-left: $fa-li-width * 1.25;`

Reply:
@MisatoTremor Feedback applied 👍